calculator.py

```diff
@@ -53,6 +61,15 @@ def get_name():
     return name
 
 
+def download_operations_history():
+    with open("operations_history.csv", "w", encoding="UTF-8") as file:
+        for operation in history_for_csv:
+            for element in operation:
+                file.write(str(element))
+                file.write(",")
+            file.write("\n")
```

Comment:
Zamiast sklejać tutaj ręcznie tutaj csvke, to użyjmy właśnie modułu csv z obiektem writer